src/main/java/com/conveyal/r5/analyst/cluster/PathResult.java

```diff
@@ -218,19 +213,15 @@ public Iteration(int totalTime) {
      * Timestamp style clock times, and rounded wait/total time, for inspection as JSON.
      */
     public static class HumanReadableIteration {
-        public String departureTime;
-        public double[] waitTimes;
-        public double totalTime;
+        public int departureTime;
+        public int[] waitTimes;
+        public int totalTime;
```

Comment:
Comment needs to be updated: `departureTime` is seconds after midnight; `waitTimes` and `totalTime` are elapsed seconds.

Reply:
This class was superseded by `PathResultSummary.Iteration` class.